src/container/styles.scss

```diff
@@ -50,12 +51,12 @@
   }
 
   // To ensure the top border/divider is visible on sticky elements which have a higher z-index
-  &.sticky-enabled:not(.refresh) {
+  &.sticky-enabled {
```

Comment:
Is this change expected?

Reply:
Nope, good catch! Must have been a merge resolution error.